core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/views/LegacyYouTubePlayerView.kt

```diff
@@ -45,6 +45,8 @@ internal class LegacyYouTubePlayerView(context: Context, attrs: AttributeSet? =
     internal var canPlay = true
         private set
 
+    var isUsingWebUi = false
+    var shouldUseWebUi = true
```

Comment:
I don't understand why we need these two variables. Isn't it enough to just use `isUsingWebUi`?

And why do we need them at all?  There was no `isUsingNativeUi` previously.

Reply:
In general I'm a bit confused by the changes in this class.

Reply:
The 2 variables are needed, Ik this is a little confusing, but I'll try to structure it to be a little easier to understand.

Reply:
Thanks.

Reply:
Well, its a little more readable now. It uses 1 variable, the reason 2 variables were used before was because `getPlayerUiController()` was being called before initialization.